honeycomb/flask-honeycomb-template.json

```diff
@@ -0,0 +1,54 @@
+{
+  "appearance": {
+    "color": "#91C7FF",
+    "icon": "fa  fa-commenting",
+    "title": "Flask-Honeycomb-Example"
+  },
+  "processing": {
+    "operations": [
+      {
+        "name": "web_hook",
+        "target": {
+          "url": "https://api.honeycomb.io/1/events/<DATASET_NAME>"
+        },
+        "customHeaders": {
+          "X-Honeycomb-Team": "<YOUR_API_KEY>"
+        },
+        "items": {
+          "status_code": "store.rookout.extracted.status_code",
+          "method": "store.rookout.extracted.method",
+          "duration": "store.rookout.extracted.duration",
+          "url": "store.rookout.extracted.url",
+          "remote_host": "store.rookout.extracted.remote_host"
+        }
+      }
+    ]
+  },
+  "aug": {
+    "location": {
+      "name": "http_server"
+    },
+    "action": {
+      "name": "script",
+      "operations": [
+        {
+          "name": "set",
+          "paths": {
+            "store.rookout.extracted": "extracted"
```

Comment:
Let's review how to best store http data in a way that will be displayed well on the FE.

How about "store.rookout.http" or "store.http"?